src/helpers/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./tuple";
```

Comment:
The location of such helpers is a very important thing, see #653 for some context. The main problem is that every module using these helpers must have reference to it's source in dependencies (either regular or peer), otherwise it would depend on hoisting with is unstable and disallowed in Yarn 2.

Reply:
Can a new repo in the TypeScriptToLua organisation be created for helper types? I'll change this PR to work with declarations coming from that repo

Reply:
If we'd decide to store it in a new package I'd rather do it in a monorepo. But I'm not really sure if that's the best option. For helpers there's an alternative - make all of these declarations global (which also would make it easier to use in declaration files), and for js compatibility just make a `lua-compat` module, which would reexport these types with JS implementation.

Reply:
Liking those ideas, what should happen with `multi.js`?

Reply:
It wouldn't be a part of TSTL, but a `lua-compat` module would reexport tstl's global helpers with a JS implementation